beta-src/src/components/ui/WDMoveControls.tsx

```diff
@@ -16,9 +15,18 @@ import {
   saveOrders,
 } from "../../state/game/game-api-slice";
 import UpdateOrder from "../../interfaces/state/UpdateOrder";
-import MoveStatus from "../../types/MoveStatus";
 import { RootState } from "../../state/store";
 import { OrderStatus } from "../../interfaces/state/MemberData";
+import OrderSubmission from "../../interfaces/state/OrderSubmission";
+
+enum Move {
```

Comment:
Maybe rename the file to `WDOrderStatusControls` and the enum to `OrderStatusButton`.?

Reply:
ok!